### PR TITLE
feat:add avatar and Badge component

### PR DESCRIPTION
**Pull request by BISINIGISHIVANI**
add avatar and badge component
Link: https://codepen.io/pys142/full/OJONymv
        https://codepen.io/pys142/full/dyZMbZb

### CODE REVIEW

File: components/Avatar/style.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.avatar{
+    border-radius:50%;
+  }
```
Comment (VishalPatil18) [inline components/Avatar/style.css:3]: A suggestion here would be that the indentation looks a bit off here and the consecutive lines. Maybe you could correct them.

File: components/Avatar/style.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    width:3rem;
+  }
+  .avatar-md{
+    width:5rem;
+  }
+  .avatar-lg{
+    width:8rem;
+  }
+  .letter-avatars{
+
```
Comment (VishalPatil18) [inline components/Avatar/style.css]: You could avoid this empty line and rather leave one empty line after each class. That helps in code readability.

File: components/Avatar/style.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  
+    background-color:green;
+    padding:0.2rem;
+  }
+  .material-icons.icon-avatar{
+    color:black;
+  }
+  .material-icons .Icon-avatar{
+    background-color:green;
+  }
```
Comment (VishalPatil18) [inline components/Avatar/style.css]: The two `icon-avatar` and `Icon-avatar` classes have the same name just they are different cased. You could change the name of anyone as it's creating a bit of confusion here. I would suggest changing the `Icon-avatar` in order to maintain consistency of small cased naming.

File: components/badge/index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    <title>Badge</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+</head>
+<body>
+    <div class="badge-1 badge-container">
+        <div  class="badge-relative">
+        <span>Text with a badge</span>
+        <span class="badge-absolute badge-sm badge-bold badge-blue badge-round">4</span>
+      </div>
```
Comment (VishalPatil18) [inline components/badge/index.html]: You can improve on the indentation here.
e.g.

```
<div class="badge-relative">
    <span>Text with a badge</span>
    <span class="badge-absolute badge-sm badge-bold badge-blue badge-round">4</span>
</div>
```

File: components/badge/index.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+        
+      <div  class="badge-relative">
+        <span>Text with Large badge</span>
+        
+        <span class="badge-absolute badge-md badge-lg badge-blue  badge-round">1</span>
+        </div>
```
Comment (VishalPatil18) [inline components/badge/index.html]: Indentation could be improved.

**Review by Prathyusha-96 — COMMENTED** [review]
code looks good.

**Review by Prathyusha-96 — APPROVED** [review]

File: components/badge/index.html [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+      </div>
+      <div  class="badge-relative">
+        <i class="material-icons fas fa-home">home</i>
+        <span class="badge-absolute badge-icon  badge-round  badge-bold badge-blue">15</span>
+        </div>
+      <div  class="badge-relative">
+        <img class="badge-round"src="https://picsum.photos/70" alt="" class="">
+        <div class="badge-absolute badge-avatar badge-round box badge-round"></div>
+        </div>
+      </div>
```
Comment (VishalPatil18) [inline components/badge/index.html]: Indentation could be improved.

File: components/badge/style.css [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+  .badge-absolute{
+    position:absolute;
+  }
+  .badge-sm{
+    top:-0.7rem;
+    left:7rem;
+  }
+  .badge-md{
+   top:-1.2rem;
+    left:8rem;
```
Comment (VishalPatil18) [inline components/badge/style.css]: Indentation could be improved.

**Review by VishalPatil18 — CHANGES_REQUESTED** [review]
- A few indentation fixes and some empty line suggestions were added.
- The `hide` button avatar is not working. You might want to check that.

Apart from these the code looks good and works fine. Well done Shivani 👍

File: components/Avatar/style.css (hunk 1 above)
Comment (BISINIGISHIVANI) [reply components/Avatar/style.css:3]: okay i resolved that

File: components/Avatar/style.css (hunk 2 above)
Comment (BISINIGISHIVANI) [reply components/Avatar/style.css]: okay i resolve that

File: components/Avatar/style.css (hunk 3 above)
Comment (BISINIGISHIVANI) [reply components/Avatar/style.css]: okay i resolve that

File: components/badge/index.html (hunk 4 above)
Comment (BISINIGISHIVANI) [reply components/badge/index.html]: okay i resolve that

File: components/badge/index.html (hunk 5 above)
Comment (BISINIGISHIVANI) [reply components/badge/index.html]: okay i resolve that

File: components/badge/style.css (hunk 7 above)
Comment (BISINIGISHIVANI) [reply components/badge/style.css]: okay i resolve that

**Review by VishalPatil18 — APPROVED** [review]
The code looks quite clean now! Great work 👍

**Comment by AnkurChunekar** [comment]
Hey, Great work. your code looks neat and clean now!
Great suggestions @VishalPatil18 !

**Review by AnkurChunekar — APPROVED** [review]

**Pull request merged**